fMRI_Tutorial/DataCheck.ipynb

```diff
@@ -0,0 +1,473 @@
+      "text/plain": [
+       "<Figure size 1080x1080 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "df  = pd.DataFrame(matfinal)\n",
+    "matdropped = df.dropna(axis=1, how='all').dropna(axis=0,how='all')\n",
+    "plt.figure(figsize=(15,15))\n",
+    "sns.heatmap(matdropped, cmap='coolwarm')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, one does not see these blank spaces anymore. One can also check how the matrice will not have NaNs anymore. \n",
+    "As discussed above, it is possible to see 3 clear diagonals in the matrix. \n",
+    "\n",
+    "\n",
+    "##### Now let's plot distribution plots."
```

Comment:
I would make this "connectivity distribution plots" and add something like: "The distribution of connectivity easily shows whether there are outliers in connectivity values in your matrix, and whether the overall distribution fits with what we expect of fMRI data (i.e. a right-skewed distribution)."

Reply:
sure!